apps/govern/components/Contracts/Votes.tsx

```diff
@@ -0,0 +1,155 @@
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Button, Flex, Space, Statistic, Table, Tooltip, Typography } from 'antd';
+import { ColumnsType } from 'antd/es/table';
+import Link from 'next/link';
+import { useMemo } from 'react';
+
+import { COLOR } from '@autonolas/frontend-library';
+
+import { useAppSelector } from 'store/index';
+import styled from 'styled-components';
+
+import { RETAINER_ADDRESS } from 'common-util/constants/addresses';
+import { CHAIN_NAMES, getBytes32FromAddress } from 'common-util/functions';
+
+const TEN_DAYS_IN_MS = 10 * 24 * 60 * 60 * 1000;
+
+const { Countdown: CountdownAntd } = Statistic;
+const { Text, Paragraph } = Typography;
+
+type MyVote = {
+  address?: string;
+  name: string;
+  chainId?: number;
+  currentWeight: number;
+  nextWeight: number;
+  isRetainer?: boolean;
+};
+
+const VotesRoot = styled.div`
+  .highlight-row {
+    background: #f2f4f9;
+  }
+`;
+
+const Countdown = styled(CountdownAntd)`
+  .ant-statistic-content {
+    font-size: 18px;
+  }
+`;
+
+const columns: ColumnsType<MyVote> = [
+  {
+    title: 'Staking contract',
+    key: 'name',
+    render: (_, record) =>
+      record.address && record.chainId ? (
+        <Space size={2} direction="vertical">
+          <Link href={`/contracts/${record.address}`}>{record.name}</Link>
+          <Text type="secondary">{CHAIN_NAMES[record.chainId] || record.chainId}</Text>
+        </Space>
+      ) : (
+        <Tooltip
+          color={COLOR.WHITE}
+          overlayStyle={{ maxWidth: '500px' }}
+          title={
+            <Paragraph className="m-0">
+              Unused staking incentives are retained unminted in the Rollover Pool. Unused staking
```

Comment:
```suggestion
              Unused staking
```

Reply:
sorry, don't get if the comment is intentional, are you suggesting a line break, or removing the text?